rest-api-cloud-client/src/main/java/com/stnetix/ariaddna/externalcloudapi/implementation/DropBoxImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.stnetix.ariaddna.externalcloudapi.implementation;
+import com.google.gson.JsonObject;
+import com.stnetix.ariaddna.externalcloudapi.cloudinterface.IAbstractCloud;
+
+import java.io.File;
+import java.net.URL;
+
+/**
+ * Created by LugovoyAV on 13.09.2017.
+ */
+public class DropBoxImpl implements IAbstractCloud {
```

Comment:
What do it this class? This class is contain only delegate method. I thing that all functionality should be move from helper class to this class.

Reply:
This class will work with the secret information of cloud(oad,save) and with file system, create directories if necessary and other.